sdk/storage/azure-storage-queue/azure/storage/queue/aio/_queue_client_async.py

```diff
@@ -419,6 +419,7 @@ def receive_messages(self, **kwargs):
             messages to retrieve from the queue, up to a maximum of 32. If
             fewer are visible, the visible messages are returned. By default,
             a single message is retrieved from the queue with this operation.
+            When retrieving these pages it is important to pass the output to `by_page()`.
```

Comment:
I feel it's a bit ambiguous to say pass the output to by_page().
we can tell them if they set messages_per_page to iterate by page, they need to call by_page() on the AsyncItemPaged, so that they can get a page iterator with which they can call next() to get the next page.